src/operators/LineSurface/CMakeLists.txt

```diff
@@ -1,144 +0,0 @@
-# DO NOT EDIT THIS FILE! THIS FILE IS AUTOMATICALLY GENERATED BY xml2cmake
```

Comment:
Just wanna make sure this was intentionally deleted.

Reply:
- [ ] look at this

Reply:
i deleted this entire operator after a discussion with the group on a telecon.

"LineSurface: lost to history unless resurrected later"

If I recall, it wasn't being built, we didn't understand if it would work or could be used given that we haven't been using it for quite some time.